server/php/settings.php

```diff
@@ -0,0 +1,54 @@
+<?php
+return [
+    'settings' => [
+        'displayErrorDetails' => true, // set to false in production
+        'addContentLengthHeader' => false, // Allow the web server to send the content-length header
+
+        // Monolog settings
+        'logger' => [
+            'name' => 'slim-app',
+            'path' => isset($_ENV['docker']) ? 'php://stdout' : __DIR__ . '/logs/app.log',
+            'level' => \Monolog\Logger::DEBUG,
+        ],
+
+        'stripe' => [
+          // You shouldn't have to touch this
+          'apiVersion' => '2019-03-14',
+
+          // Update this path if you want to move your public folder
+          'staticDir' => __DIR__ . '/../../public/',
+
+          // Find them at https://dashboard.stripe.com/account/apikeys
+          'publishableKey' => '',
+          'secretKey' => '',
```

Comment:
We shouldn't have the secret key in a file that is committed to git. Can we made take this approach? https://medium.com/@hfally/how-to-create-an-environment-variable-file-like-laravel-symphonys-env-37c20fc23e72

Reply:
Good point, added ini file, see e131cf48cf1dc9d9c3113821af789c6497c285e6